CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.13](https://github.com/brave/brave-ios/releases/tag/v1.13)
+ 
+ - Implement Brave Rewards and Ads. ([#1920](https://github.com/brave/brave-ios/issues/1920))
+ - Added new preview menu on iOS13. ([#1499](https://github.com/brave/brave-ios/issues/1499))
+ - Added bookmarklet support. ([#1119](https://github.com/brave/brave-ios/issues/1119))
+ - Updated text size and colours on new tab stats to match desktop. ([#1641](https://github.com/brave/brave-ios/issues/1641))
```

Comment:
Change `colours` --> `color` (was asked to use color for desktop so we should keep things consistent)